constants/util.go

```diff
@@ -86,5 +90,21 @@ var (
 				CreateDir,
 			},
 		},
+		Flye: {
+			Name:             Flye,
+			DHubURL:          "docker.io/nanozoo/flye", // https://github.com/fenderglass/Flye
+			OutputDir:        FlyeOut,
+			AssemblyFileName: "assembly.fasta",
+			Comm: func(threads int) []string {
```

Comment:
Is this off of an old version of master? I thought this changed to take in a `*parser_config.Config`, which contains the threads.

Reply:
<img width="981" alt="Screen Shot 2020-05-23 at 15 15 41" src="https://user-images.githubusercontent.com/18152652/82740700-749d8180-9d08-11ea-90b4-6ba88d51534a.png">

Reply:
Oh, I just saw the commits, this is an old branch we've both contributed to for adding Flye...